restfulpy/tests/test_commit_decorator.py

```diff
@@ -72,3 +88,31 @@ def test_rollback(self):
         with self.given('Raise exception', verb='ERROR', url='/'):
             assert response.status == 500
 
+    def test_commit_on_raise_http_success(self):
+        with self.given(
+            'Testing the operation of commit decorator on raise 2xx',
+            verb='SUCCESS',
```

Comment:
No need to mention `verb=` and `url=` if you preserve the order. check python's documentation.